charts/apache-activemq/values.yaml

```diff
@@ -0,0 +1,2 @@
+image: apache/activemq-classic:5.18.2
+loadGeneratorImage: bmedora/activemq-loadgen:test123
```

Comment:
We should use the image built by this repo's ci/cd

Reply:
There will be a follow up PR, as noted in the description.